### ISSUE TITLE
Add jacoco package to the default packages to be deferred

### ISSUE THREAD

**Issue by GoogleCodeExporter** [nkxiaochuan/powermock#458]
```
What steps will reproduce the problem?
1. Use PowerMock with jacoco offline instrumentation
2. In the test result, one can see errors like :
java.nio.channels.OverlappingFileLockException
        at sun.nio.ch.FileChannelImpl$SharedFileLockTable.checkList(FileChannelImpl.java:1166)
        at sun.nio.ch.FileChannelImpl$SharedFileLockTable.add(FileChannelImpl.java:1068)
        at sun.nio.ch.FileChannelImpl.lock(FileChannelImpl.java:824)
        at java.nio.channels.FileChannel.lock(FileChannel.java:860)
        at org.jacoco.agent.rt.internal_9dd1198.output.FileOutput.openFile(FileOutput.java:69)
        at org.jacoco.agent.rt.internal_9dd1198.output.FileOutput.writeExecutionData(FileOutput.java:53)
        at org.jacoco.agent.rt.internal_9dd1198.Agent.shutdown(Agent.java:143)
        at org.jacoco.agent.rt.internal_9dd1198.Agent$1.run(Agent.java:60)
3.

What is the expected output? What do you see instead?


What version of the product are you using? On what operating system?
PowerMock 1.5 + Jacoco 0.6.3

Please provide any additional information below.

The problem is that jacoco class 
org.jacoco.agent.rt.internal_9dd1198.output.FileOutput registers a shutdown 
hook in the JVM which should normally execute only once per JVM.
With PowerMock, jacoco classes are loaded several times because of the 
MockClassLoader and thus the shutdownhook is registered and executed several 
times in parallel when the JVM shuts down, leading to the above 
OverlappingFileLockException.

One could add @PowerMockIgnore("org.jacoco.agent.rt.*") to each test class but 
this is annoying.
I see that there's already a list of packages to ignore by default, in 
org.powermock.core.classloader.MockClassLoader.packagesToBeDeferred

Please add "org.jacoco.agent.rt.*" to this list.

Also, it would be nice to be able to add such ignores globally with a system 
property !
```

Original issue reported on code.google.com by `sylvain....@gmail.com` on 13 Sep 2013 at 7:43
